### ISSUE TITLE
Less clutter in keyboard layout message

### ISSUE THREAD

**Issue by Vladimir-csp** [google/xsecurelock#95]
Layout indicator in auth box is sometimes a bit messy.
For `us` layout it is ok, the message just says: `Keyboard: English (US)`
But for `ru` layout it says: `Keyboard: Russian, Scroll Lock, Group 2`

The second field here is not helpful. Is this message a reaction to scroll lock LED state, or to `grp_led:scroll` option in xkb? Can the actual state of scroll lock be tracked, not just LED?
Can scroll lock mess with password entry in principle? If not, then I doubt that scroll lock tracking is at all needed in the auth box.

The third field (`Group 2`) can either be dropped completely, or shown if there are more than 2 layouts.

**Comment by divVerent** [comment]
Yes, it is correct that we're showing the LED status. It currently shows all LEDs, as the Xkb API does not actually reveal information about what each LED means.

There is an alternate API that reveals Caps Lock but not Num Lock, by looking at the modifiers returned by XkbGetState. This, of course, has its own issues, as Num Lock can affect password typing a lot, especially on laptops.

**Comment by divVerent** [comment]
Can you try with this commit and XSECURELOCK_SHOW_LOCKS_AND_LATCHES=1 set? I don't quite like it yet as I have not found a good source for the names yet, but this otherwise is probably closer to the minimum info that should be shown.

**Comment by Vladimir-csp** [comment]
Num lock is shown as `Mod2`, caps lock is shown as just `Lock`.
But when those are not engaged, layouts themselves are shown cleanly.

**Comment by Barbarossa93** [comment]
Perhaps I should make another issue for this, but I think that having an option to completely hide what the current keyboard layout is along with activated locks would be great for those of us who prefer minimal setups

**Comment by andys8** [comment]
> having an option to completely hide what the current keyboard layout is along with activated locks would be great for those of us who prefer minimal setups

Would be really cool :) Mine shows "Keyboard: German (no dead keys)"
